server/protocols/jmap/pom.xml

```diff
@@ -274,6 +274,21 @@
                     <artifactId>bcpkix-jdk15on</artifactId>
                 </dependency>
                 <dependency>
+                    <groupId>org.javatuples</groupId>
```

Comment:
I understand you tried to fix the file ordering, but you failed at that because the current state is too bad.
Please make another commit to reorder everything properly.